config/initializers/env.rb

```diff
@@ -34,6 +34,10 @@
   validation_mail_to: ENV['GALLERY_VALIDATION_MAIL_TO']
 )
 
+Rails.application.config.x.exhibitions = OpenStruct.new(
+  host: ENV['EXHIBITIONS_HOST'] ||'https://' + (ENV['HTTP_HOST'] || 'www.europeana.eu'),
+  annotation_creator_name: ENV['EXHIBITION_ANNOTATION_CREATOR_NAME'] || 'Europeana.eu Exhibition'
```

Comment:
If this env var is going here, let's name it `EXHIBITIONS_ANNOTATION_CREATOR_NAME` (plural)